src/types/package.go

```diff
@@ -16,19 +16,20 @@ type ZarfPackage struct {
-	Authors           string `json:"authors,omitempty" jsonschema:"description=List of package authors (including contact info)"`
-	Documentation     string `json:"documentation,omitempty" jsonschema:"description=Link to package documentation when online"`
-	Source            string `json:"source,omitempty" jsonschema:"description=Link to package source code when online"`
-	Vendor            string `json:"vendor,omitempty" jsonschema_description:"Name of the distributing entity, organization or individual."`
-	AggregateChecksum string `json:"aggregateChecksum,omitempty" jsonschema:"description=Checksum of a checksums.txt file that contains checksums all the layers within the package."`
+	Name              string         `json:"name" jsonschema:"description=Name to identify this Zarf package,pattern=^[a-z0-9\\-]+$"`
+	Description       string         `json:"description,omitempty" jsonschema:"description=Additional information about this package"`
+	Version           string         `json:"version,omitempty" jsonschema:"description=Generic string set by a package author to track the package version (Note: ZarfInitConfigs will always be versioned to the CLIVersion they were created with)"`
+	URL               string         `json:"url,omitempty" jsonschema:"description=Link to package information when online"`
+	Image             string         `json:"image,omitempty" jsonschema:"description=An image URL to embed in this package (Reserved for future use in Zarf UI)"`
+	Uncompressed      bool           `json:"uncompressed,omitempty" jsonschema:"description=Disable compression of this package"`
+	Architecture      string         `json:"architecture,omitempty" jsonschema:"description=The target cluster architecture for this package,example=arm64,example=amd64"`
+	YOLO              bool           `json:"yolo,omitempty" jsonschema:"description=Yaml OnLy Online (YOLO): True enables deploying a Zarf package without first running zarf init against the cluster. This is ideal for connected environments where you want to use existing VCS and container registries."`
+	Authors           string         `json:"authors,omitempty" jsonschema:"description=List of package authors (including contact info)"`
+	Documentation     string         `json:"documentation,omitempty" jsonschema:"description=Link to package documentation when online"`
+	Source            string         `json:"source,omitempty" jsonschema:"description=Link to package source code when online"`
+	Vendor            string         `json:"vendor,omitempty" jsonschema_description:"Name of the distributing entity, organization or individual."`
+	AggregateChecksum string         `json:"aggregateChecksum,omitempty" jsonschema:"description=Checksum of a checksums.txt file that contains checksums all the layers within the package."`
+	ConnectStrings    ConnectStrings `json:"connectStrings,omitempty" jsonschema:"description=List of connection strings for the package"`
```

Comment:
Can we add this to the `DeployedPackage` type instead? (having this here will make people think it is a thing for them to specify since this type is user-facing)

Reply:
Sounds good made that change!